badge-maker/index.d.ts

```diff
@@ -4,6 +4,10 @@ interface Format {
   labelColor?: string
   color?: string
   style?: 'plastic' | 'flat' | 'flat-square' | 'for-the-badge' | 'social'
+  logo?: string
```

Comment:
At this layer, lets call this param `logoBase64` instead of just `logo` so that it is not ambiguous with named logos when we add that.

Reply:
Just thinking about this again.. in the endpoint schema, we expose `logoSvg` (which is literally an SVG as a string) and `namedLogo`. I wonder if we should just expose `logoSvg` here and encode it for the user to keep the package API consistent with the endpoint schema. Base64 is useful in URLs, but we don't have that constraint here. The only downside is we can't accept encoded PNG logos.

I am being quite indecisive here :thinking:

Reply:
@chris48s so what do you think, keep it `logo` or change it to `logoBase64 `? I personally think that `logo` is better for consistence , but in future it could also work as `logo` property in other places

Reply:
Meh. Lets roll with `logoBase64` for now. It is a quick easy way to expose _some_ way for package users to use logos. We can add `logoSvg` and/or `namedLogo` at another time. I don't think any of those concepts should be called just `logo`.